Makefile

```diff
@@ -190,7 +191,7 @@ docker-push-check-img:
 ##@ Test
 
 .PHONY: test
-test: build manifests generate fmt vet verify-licenses gotest integration-tests integration-tests-v2 ## Run unit tests and integration tests
```

Comment:
I believe `lint` is preferable to `fmt` since developers usually expect tests to have no side effects. Please note that the `lint` target also executes `vet`.